src/core/constraint.jl

```diff
@@ -37,11 +66,61 @@ end
 
 ""
 function constraint_active_branch(pm::_PMs.AbstractPowerModel,  i::Int, nw_1::Int, nw_2::Int)
-    if haskey(_PMs.ref(pm, nw_1, :branch_damaged), i)
+    if haskey(_PMs.ref(pm, nw_1, :damaged_branch), i)
         z_branch_1 = _PMs.var(pm, nw_1, :z_branch, i)
         z_branch_2 = _PMs.var(pm, nw_2, :z_branch, i)
 
         JuMP.@constraint(pm.model, z_branch_2 >= z_branch_1)
     end
 end
 
+"Load delivered at each node must be greater than or equal the previous time period"
+function constraint_increasing_load(pm::_PMs.AbstractPowerModel,  i::Int, nw_1::Int, nw_2::Int)
+    if haskey(_PMs.ref(pm, nw_1, :damaged_branch), i)
+        z_demand_1 = _PMs.var(pm, nw_1, :z_demand, i)
+        z_demand_2 = _PMs.var(pm, nw_2, :z_demand, i)
+
+        JuMP.@constraint(pm.model, z_demand_2 >= z_demand_1)
+    end
+end
+
+
+"on/off constraint for generators connected to damaged buses"
+function constraint_gen_bus_connection(pm::_PMs.AbstractPowerModel, n::Int, c::Int, gen_id::Int, bus_id::Int, pmin, pmax, qmin, qmax)
```

Comment:
This constraint does not require any data about the generator and can simply be,
```
z_bus = PMs.var(pm, n, :z_bus, bus_id)
z_gen = PMs.var(pm, n, :z_gen, gen_id)
JuMP.@constraint(pm.model, z_gen <= z_bus)
```
A similar pattern can be used for all other component types and it does not need to be specialized to any particular PowerModel type.

Reply:
Again, the issue of undamaged generators attached to damaged buses.

Reply:
Again, same note as above.